### ISSUE TITLE
Actionable Element: Tab

### ISSUE THREAD

**Issue by Ibabalola** [Capgemini/dcx-react-library#629]
“When trying to access the ‘Add products from favourites’ tab I could not do so using
keyboard. This means keyboard users would be unable to use this option. Keyboard users
are unable to access areas of the site that mouse users can.”

**Expected behavior**
Ensure that all content is accessible with a keyboard, and does not rely on the use of a
mouse. The tabindex attribute will need to be removed in order to fix the issue. This will put
the link in the tab order and all users will be able to access it.

**Screenshots**
<img width="339" alt="Screenshot 2024-04-15 at 16 19 26" src="https://github.com/Capgemini/dcx-react-library/assets/11960286/0323fa88-46de-4df8-bd0b-aa89afd2b838">

**Desktop (please complete the following information):**
 - Browser: Chrome

**Comment by daniele-zurico** [comment]
closed by https://github.com/Capgemini/dcx-react-library/pull/630
